tests/src/JIT/HardwareIntrinsics/X86/Avx/Avx_ro.csproj

```diff
@@ -44,14 +44,6 @@
     <Compile Include="DuplicateEvenIndexed.Double.cs" />
     <Compile Include="DuplicateEvenIndexed.Single.cs" />
     <Compile Include="DuplicateOddIndexed.Single.cs" />
-    <Compile Include="Extract.Byte.1.cs" />
```

Comment:
I wonder  if just commenting these out would be better, since we will need to add them back once the changes flow to CoreFX and back again....

Reply:
Ah, yes, but I already removed them 😄

Reply:
Might be good to fix it, so we don't have +10k/-10k diffs for the same code in such a brief period...